cards3_0.py

```diff
@@ -0,0 +1,39 @@
+import class_card
+from class_card import Card
+import class_deck
+from class_deck import Deck
+import class_Hand
+from class_Hand import Hand
+
+class Unpritabale_Card(Card):
```

Comment:
Typo `Unpritabale_Card`
```suggestion
class Unpritabale_Card(Card):
```